OpenEXR/IlmImf/ImfDwaCompressor.cpp

```diff
@@ -742,7 +742,7 @@ DwaCompressor::LossyDctDecoderBase::execute ()
 
     for (int i = 0; i < _SSE_ALIGNMENT; ++i)
     {
-        if (((size_t)(rowBlockHandle + i) & _SSE_ALIGNMENT_MASK) == 0)
+        if ((reinterpret_cast<size_t>(rowBlockHandle + i) & _SSE_ALIGNMENT_MASK) == 0)
```

Comment:
a static_cast to uintptr_t would be idiomatic here

Reply:
God to know... thx. Done. Learnt something today. ;)